README.md

```diff
@@ -72,4 +72,13 @@ convivaConfig.setCustomData(customMapOfKeyValuePairs);
 
 ```
 
+#### Consecutive playback
+	
+If you want to use the same player instance for multiple playback call `player.unload()` before loading a new source.
```

Comment:
```suggestion
If you want to use the same player instance for multiple consecutive playback call `player.unload()` before loading a new source.
```

Reply:
similar to the iOS feedback, I would like to see this documentation inform the user how to update metadata between playback sessions